client/lib/signup/flow-controller.js

```diff
@@ -221,6 +226,20 @@ assign( SignupFlowController.prototype, {
 		return this._flow.destination;
 	},
 
+	_getStoredDependencies() {
+		const requiredDependencies = this._flow.steps.reduce( ( current, stepName ) => {
+			if ( ! steps[ stepName ] || ! steps[ stepName ].providesDependencies ) {
```

Comment:
lodash `get` might make this look easier on the eyes. `get( steps, [ stepName, 'providesDependencies' ] )`

```
const providesDependencies = get( steps, [ stepName, 'providesDependencies' ] );
return providesDependencies ? current.concat( providesDependencies ) : current;
```